background_jobs/ReadMe.md

```diff
@@ -0,0 +1,21 @@
+# ARC background processing framework
+
+## Overview
+
+The goal of this submodule is to provide simple and convenient way to schedule repetetive tasks to be performed on ARC.
+The current implementation contains only ARC blocks clean up with frequency given in configuration file.
+
+## Key features
+- easy configuration using config.yaml file
+- no dependencies
+- structured logs output using zerolog
+- extendable layout for other scenarios
+
+## System requirements
+- `go version go1.21.1`
```

Comment:
The required go version is defined in `go.mod`
If we state that here, we have to always update if we upgrade go version. So IMO we can remove this system requirement

Reply:
Well, it's an established practice to list all the requirements for software to work correctly with it's version. It may or may not with other versions.

Reply:
Plus we will have to gradually upgrade all our code to latest versions, 
Updating to most recent version is mandatory for security, performance and support for additional features.